src/SourceFile.js

```diff
@@ -138,7 +138,12 @@ class SourceFile extends DirItem {
                 for (const ir of irs) {
                     // find the rules that are appropriate for this intermediate representation and then apply them
                     const rules = this.filetype.getRules().filter((rule) => rule.getRuleType() === ir.getType());
-
+                    if (!(this.project.options.opt.quiet) && this.project.options.opt.progressInfo) {
+                        rules.forEach(function(ru){
+                            logger.info("Checking rule  : " + ru.name);
```

Comment:
I think this should probably be logger.debug instead. That's a lot of information for each file...
Checking the file names as it goes is probably fine, but each rule for each file will be a LOT of output.

Reply:
Updated a release notes.
and looks like I need to set logger.level in somewhere to print logger.debug message. 
I updated to print filepath  only when the `progressinfo` option is enabled. Please check my change again.